semgrep/semgrep/ignores.py

```diff
@@ -1,121 +1,228 @@
-"""
-Handles ignoring Semgrep findings via inline code comments
-
-Currently supports ignoring a finding on a single line by adding a
-`# nosemgrep:ruleid` comment (or `// nosemgrep:ruleid`).
-
-To use, create a RuleMatchMap, then pass it to process_ignores().
-"""
-from re import sub
-from typing import List
-from typing import Sequence
-from typing import Tuple
+import fnmatch
```

Comment:
Now that this filtering happens in semgrep, can we get rid of it in semgrep-action? If we're holding off, should comment that with a TODO and also indicate which parts are duplicated. Definitely err on the side of too much explanation in this case

Reply:
For other reviewers: https://www.diffchecker.com/kLZRpCYn

Reply:
Are my added comments explaining the duplication enough, or are they too vague?

Reply:
They look good! I would additionally add a TODO for removing semgrep-ignore from the action.